src/components/parameters/card_ids.yaml

```diff
@@ -0,0 +1,8 @@
+in: query
+name: card_ids
```

Comment:
why do you need card_ids plural?

Reply:
its a query param for filtering. Say in spend by merchant tab admin wants to see spend from specific cards

In design, few filterns supported single value and few supported multiple values. So I am supporting multiple values for all in API. Web-app can still send single value in a list